### ISSUE TITLE
Integration of probable/antigen cases for Slovakia

### ISSUE THREAD

**Issue by CSSEGISandData** [CSSEGISandData/COVID-19#4925]
Hello all,

In #4924, we have integrated probable/antigen cases for Slovakia from [their dashboard](https://korona.gov.sk/koronavirus-na-slovensku-v-cislach/#covid-aut-nasledujuci-pondelok). These cases meet our definition of probable cases consistent with the CDC probable case definition. We have used data from the [department's GitHub repository](https://github.com/Institut-Zdravotnych-Analyz/covid19-data/blob/main/DailyStats/OpenData_Slovakia_Covid_DailyStats.csv) to back distribute these probable cases and thus avoid any artificial spike in the dataset.

**Comment by kurcam** [comment]
I don't understand this politics! Did you integrated probable/antigen cases also for another countries? For example Slovakia's nearest neighbour Czechia (https://onemocneni-aktualne.mzcr.cz/covid-19) has actual 2 062 064 confirmed cases (PCR tests) and 299 645 antigen cases. But you are publishing 2 044 018 cases for November 24th. It seems that these data are not comparable data and all your statistics is worthless.

**Comment by CSSEGISandData** [comment]
@kurcam thank you for your comment. The 299,645 cases for the Czech Republic lists the cumulative number of cases among those aged 65 and older and not the total number of positive antigen tests. The confirmed case total (2,062,064) is inclusive of PCR and antigen positive cases (See [Czech README](https://onemocneni-aktualne.mzcr.cz/covid-19--napoveda0)). Integrating probable cases for Slovakia allows the comparisons between countries to be more fair, as in this specific case the total number of cases for Slovakia and the Czech Republic now fulfill the same criteria.

**Comment by kurcam** [comment]
Yes, you are right. My mistake, that I read the titles of data above (total PCR tests made and total AG test made). So are you sure that all countries have published all confirmed cases together with probable/antigen cases ? It is a big difference with regard to Slovakia (where AG tests were widely used.
